### PR TITLE
modify terminated-pod-gc-threshold explanation (#15272)

### PR DESCRIPTION
**Pull request by hase1128**
I modify terminated-pod-gc-threshold explanation

detail information is here.
https://github.com/kubernetes/website/issues/15272

### CODE REVIEW

**Comment by kbhawkey** [comment]
Hello @hase1128 . Thanks for submitting this PR.
I am looking into the change. In the meantime, would it be possible to remove the issue number from the commit line?
Page preview:
https://deploy-preview-17183--kubernetes-io-master-staging.netlify.com/docs/concepts/workloads/pods/pod-lifecycle/

File: content/en/docs/concepts/workloads/pods/pod-lifecycle.md [hunk 1]
```diff
@@ -278,8 +278,9 @@ once bound to a node, a Pod will never be rebound to another node.
 
 In general, Pods do not disappear until someone destroys them. This might be a
 human or a controller. The only exception to
-this rule is that Pods with a `phase` of Succeeded or Failed for more than some
-duration (determined by `terminated-pod-gc-threshold` in the master) will expire and be automatically destroyed.
+this rule is that Pods will be automatically destroyed if the number of Pods 
+with a `phase` of Succeeded or Failed surpasses a configurable threshold 
+(determined by `terminated-pod-gc-threshold` in the master).
```
Comment (kbhawkey) [inline content/en/docs/concepts/workloads/pods/pod-lifecycle.md]: @hase1128 , I like the new update -- here is a possible tweak, starting with line 279:
In general, Pods remain until a human or controller process explicitly removes them.
However, Pods will be automatically destroyed if the number of terminated Pods, with a `phase` of Succeeded or Failed, surpasses a configurable threshold (determined by `terminated-pod-gc-threshold` in the master).

Comment (sftim) [reply content/en/docs/concepts/workloads/pods/pod-lifecycle.md]: My suggestion for the final sentence:
```markdown
Above a configured number of terminated Pods (with a phase of `Succeeded`
or `Failed`), the control plane cleans up some terminated Pods. This avoids
a resource leak as Pods as created and terminated over time. The clean up
threshold is based on the setting of `terminated-pod-gc-threshold` in the
kube-controller-manager.
```

Comment (kbhawkey) [reply content/en/docs/concepts/workloads/pods/pod-lifecycle.md]: @hase1128, do you think you could incorporate @sftim 's suggestion into your PR?
@sftim, your suggestion offers a bit more detail, though, the first sentence seems more complicated.
What about starting the sentence as:
The control plane cleans up terminated Pods (with a ...) when the number of ... exceeds the configured threshold.

Also did you mean,
`This avoids a resource leak as Pods are created and terminated over time`?

Comment (hase1128) [reply content/en/docs/concepts/workloads/pods/pod-lifecycle.md]: @kbhawkey @sftim 

I incorporate your comments. How about the following explanation?
```
In general, Pods remain until a human or controller process explicitly removes them.
The control plane cleans up terminated Pods (with a phase of `Succeeded` or
`Failed`), when the number of Pods exceeds the configured threshold
(determined by `terminated-pod-gc-threshold` in the kube-controller-manager).
This avoids a resource leak as Pods are created and terminated over time.
```

**Comment by kbhawkey** [comment]
Thanks!
/lgtm

**Comment by hase1128** [comment]
@kbhawkey 
Thank you for the comment and lgtm label.
Would you also add `approved` label?
I think this PR can be merged.

**Pull request merged**